app/(auth)/welcome/index.tsx

```diff
@@ -1,11 +1,53 @@
-import { Link } from 'expo-router'
-import { Text, View } from 'react-native'
+import { useState } from 'react'
+import { View } from 'react-native'
+import PagerView from 'react-native-pager-view'
+import { ButtonLink, CarouselIndicators, Welcome } from '../../../components'
+
+const url = '../../../assets/images/'
+
+const imageSource = {
```

Comment:
elimina los objetos, en su lugar declara las las importaciones en variables directas : 
```suggestion
const imageSource = require('asset')
```